### DISCUSSION TITLE
PixArt-ControlNet is available!!

### DISCUSSION THREAD

**Discussion by lawrence-cj** [PixArt-alpha/PixArt-alpha#65] [Announcements]
https://github.com/PixArt-alpha/PixArt-alpha/tree/feat/pixart-ctrlnet-public

**Comment by rainsoulsrx** [comment]
> https://github.com/PixArt-alpha/PixArt-alpha/tree/feat/pixart-ctrlnet-public

Hi， if I want to train controlnet, how should I organize mty train datasets?

**Comment by greasebig** [comment]
can this controlnet be used in comfyui?
